### PR TITLE
Add participants

### LINKED ISSUE #746: Refactor sendUserData and addPersonToEvent together into one function.
Currently in participants.py we have one function to add event participants when through kiosk (sendUserData) and another for adding event participants all other ways (addPersonToEvent). Combine them into one function.

### PR DESCRIPTION
**Pull request by doucoureb**

### CODE REVIEW

File: app/controllers/events/routes.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def signinEvent():
         elif bnumber[0].upper() != "B":
             return "", 500
     try:
-        kioskUser, userStatus = sendUserData(bnumber, eventid, programid)
+        kioskUser, userStatus = sendUserData(userObj.bnumber, eventid, programid)
```
Comment (AndersonStettner) [inline app/controllers/events/routes.py]: This should have stayed `bnumber`.

**Review by AndersonStettner — CHANGES_REQUESTED** [review]

File: app/logic/participants.py [hunk 2]
```diff
@@ -64,34 +70,10 @@ def checkUserRsvp(user,  event):
...
-            return "already in"
-    except Exception as e:
-        print(e)
-        return False
-
-    return True
+#         if event.isPast:
+#             if not volunteerExists:
+#                 eventHours = getEventLengthInHours(event.timeStart, event.timeEnd, event.startDate)
+#                 EventParticipant.create(user = user, event = event, hoursEarned = eventHours)
```
Comment (BrianRamsay) [inline app/logic/participants.py]: Remove the comments that aren't being used, please

**Review by BrianRamsay — APPROVED** [review]

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
As part of this refactor we should rename `sendUserData` to a meaningful name.

**Comment by BrianRamsay** [comment]
I did this another way in PR #973

**Pull request closed without merging**